Config.uk

```diff
@@ -0,0 +1,17 @@
+menuconfig LIBGEOS
+	bool "libgeos - a C/C++ library for GIS computational geometry"
+	default n
+	select LIBCXX
+	select CXX_THREADS
+
+
+if LIBGEOS
+    config LIBGEOS_CLI
+    bool "Provide the GeosOp command-line interface as main"
+    default n
+
+    config LIBGEOS_UNITTEST
+    bool "Provide the libgeos unit tests as main"
+    default n
```

Comment:
These look like they should be mutually exclusive, since they both provide the main function.

Reply:
True, will change to a prompt.